### ISSUE TITLE
Unexpected inline handling of noqa comments

### ISSUE THREAD

**Issue by PeterJCLaw** [schlamar/flake8-todo#14]
Thanks for creating this plugin, it's really useful to limit TODO comments and the like.

It looks like the plugin is however doing its own handling of noqa comments, which is unfortunately both incompatible with how `flake8` normally handles such comments and means that other usage of the `T000` error code doesn't work in various ways.

For example here's some code which should report `T000` but does not:
``` python
import os # noqa: F401 # TODO: fix this
```

It also means that `flake8-noqa` cannot be used in conjunction with this plugin to ignore TODOs which are desired for some reason (more context at https://github.com/plinss/flake8-noqa/issues/29):
``` python
# TODO: make this better # noqa: T000
```
ends up reporting a `NQA102` error even though there ought to be an error for the `noqa` comment to hide.

**Comment by PeterJCLaw** [comment]
I'd be happy to put together a pull request which addressed this if you're open to that.
